addons/payment/wizards/payment_onboarding_wizard.py

```diff
@@ -96,7 +90,6 @@ def add_payment_methods(self):
             if self.payment_method == 'paypal':
                 new_env.ref('payment.payment_provider_paypal').write({
                     'paypal_email_account': self.paypal_email_account,
-                    'paypal_seller_account': self.paypal_seller_account,
                     'paypal_pdt_token': self.paypal_pdt_token,
                     'state': 'enabled',
```

Comment:
I think we forgot to publish the provider when the user is setting it up through the onboarding payment wizard. Could you make the change here and a few lines below (if the `payment_method` is "manual")?
```suggestion
                    'state': 'enabled',
                    'is_published': True
```